pkgs/data/themes/orchis/default.nix

```
@@ -0,0 +1,35 @@
+{ accentColor ? "blue"
+, lib, stdenv, fetchFromGitHub, gtk3, gnome-themes-extra, gtk-engine-murrine }:
+
+stdenv.mkDerivation rec {
+  pname = "orchis";
+  version = "2021-01-22";
+
+  src = fetchFromGitHub {
+    repo = "Orchis-theme";
+    owner = "vinceliuice";
+    rev = "${version}";
+    sha256 = "1m0wilvrscg2xnkp6a90j0iccxd8ywvfpza1345sc6xmml9gvjzc";
+  };
+
+  nativeBuildInputs = [ gtk3 ];
+
+  buildInputs = [ gnome-themes-extra ];
+
+  propagatedUserEnvPkgs = [ gtk-engine-murrine ];
+
+  dontPatch = true;
+  dontConfigure = true;
+  dontBuild = true;
+
+  preInstall = "mkdir -p $out/share/themes";
+  installPhase = "bash install.sh -d $out/share/themes" + lib.optionalString (accentColor != "blue") " -t ${accentColor}";
```

```suggestion
  preInstall = ''
    mkdir -p $out/share/themes
  '';

  installPhase = ''
    runHook preInstall
    bash install.sh -d $out/share/themes -t ${accentColor}
    runHook postInstall
  '';
```

Does that work when we always specify the accent color?

---

I think yes. The default is "blue" and the script works the same whether there's `-t blue` present or not.